### ISSUE TITLE
Workflow failed to build

### ISSUE THREAD

**Issue by w-sx** [nvdaaddons/AddonTemplate#39]
I cloned this project and applied it to my NVDA Addon, and enabled the workflow, I received an error notification after `git push --tags`.

```
File "/home/runner/work/virtualKeyboard/virtualKeyboard/sconstruct", line 213
result_brailleTables.append(f"displayName = {buildVars.brailleTables[table]["displayName"]}")
SyntaxError: f-string: unmatched '['
Error: Process completed with exit code 2.
```

This statement works fine on my local Windows platform, I don't know why it reports an error here. Anyway, I fixed the above error. I received another error notification after `git push --tags`.

```
scons: Reading SConscript files ...
Python 3.10 or greater required, but you have Python 3.9.19
Error: Process completed with exit code 2.
```

After fixing the above error, the NVDA Addon built correctly.

In summary, the specific suggestions for modification are as follows:

In the `build_addon.yml` file
	`python-version: 3.9`
	It is recommended to change to
	`python-version: 3.11`

In the `sconstruct` file
	`result_brailleTables.append(f"displayName = {buildVars.brailleTables[table]["displayName"]}")`
	It is recommended to change to
	`result_brailleTables.append(f"displayName = {buildVars.brailleTables[table]['displayName']}")`

These are just a few minor modifications, I have already modified them, and I would be happy to submit a PR to fix this issue.

**Comment by josephsl** [comment]
Hello, thank you for reporting this, and offering a PR for it (go ahead). Thanks.
